Userland/Libraries/LibGfx/BitmapFont.cpp

```diff
@@ -262,19 +257,18 @@ ALWAYS_INLINE int BitmapFont::unicode_view_width(T const& view) const
     int longest_width = 0;
 
     for (u32 code_point : view) {
-        if (code_point == '\n' || code_point == '\r') {
+        if (code_point == '\n' || code_point == '\r') [[unlikely]] {
```

Comment:
This looks like pure cargo-culting unless you have numbers to justify it :)

Reply:
I didn't do measurements for each single change, so I don't have the numbers. If you don't want it, I'll remove it. :^)

Reply:
Yeah I think we're better off without these. Let's only do this sort of thing if we *know* that it makes a difference. :)